src/Kernel/UnhandledError.class.st

```diff
@@ -55,5 +55,5 @@ UnhandledError >> exception: anError [
 { #category : #testing }
 UnhandledError >> isResumable [
 	
-	^ false
+	^ true
```

Comment:
It deserves a good comment why it must be resumable.
There is #resumeUnchecked: method to resume any exception. Would not it be better to use it explicitly for your goal?

Now it feels very odd to have false for Error>>isResumable and true for UnhandledError>>isResumable.

Reply:
Using #resumeUnchecked: w/ the changes to #defaultAction and #raiseUnhandledError would be sufficient. It could also be worked around using #resignalAs:. 

I'll revert this method and add a test for the return value of #defaultAction.

Reply:
After a bit of discussion w/ @martinmcclure, we decided it may be appropriate to have UnhandledError reusability reflect that of the underlying Exception. The changes in #defaultAction and #raiseUnhandledError allow the resumption value of UnhandledError to serve as the resumption value of the original #signal.

In the event that the underlying Exception is not resumable, an IllegalResumeAttempt is signal.

This would allow us to avoid the use of #resumeUnchecked: while making UnhandledError more generally useful.